src/fenicsxconcrete/experimental_setup/simple_beam.py

```diff
@@ -0,0 +1,139 @@
+from fenicsxconcrete.experimental_setup.base_experiment import Experiment
+from fenicsxconcrete.helper import Parameters
+import dolfinx as df
+from mpi4py import MPI
+import numpy as np
+import ufl
+from petsc4py.PETSc import ScalarType
+from fenicsxconcrete.unit_registry import ureg
+import pint
+from fenicsxconcrete.boundary_conditions.bcs import BoundaryConditions
+from fenicsxconcrete.boundary_conditions.boundary import plane_at, point_at, line_at
+
+class SimpleBeam(Experiment):
+    """Sets up a simply supported beam, fix on the left
+
+    Attributes:
+        see base class
+    """
+
+    def __init__(self, parameters: dict[str, pint.Quantity]):
+        """defines default parameters, for the rest, see base class"""
+
+        # initialize default parameters for the setup
+        default_p = Parameters()
+        default_p['degree'] = 2 * ureg('')  # polynomial degree
+
+        # updating parameters, overriding defaults
+        default_p.update(parameters)
+
+        super().__init__(default_p)
+
+    def setup(self):
+        """defines the mesh for 2D or 3D"""
+
+        if self.p['dim'] == 2:
+            self.mesh = df.mesh.create_rectangle(comm=MPI.COMM_WORLD,
+                                                 points=[(0.0, 0.0),
+                                                         (self.p['length'], self.p['height'])],
+                                                 n=(self.p['num_elements_length'],
+                                                    self.p['num_elements_height']),
+                                                 cell_type=df.mesh.CellType.quadrilateral)
+        elif self.p['dim'] == 3:
+            self.mesh = df.mesh.create_box(comm=MPI.COMM_WORLD,
+                                           points=[(0.0, 0.0, 0.0),
+                                                   (self.p['length'], self.p['width'], self.p['height'])],
+                                           n=[self.p['num_elements_length'],
+                                              self.p['num_elements_width'],
+                                              self.p['num_elements_height']],
+                                           cell_type=df.mesh.CellType.hexahedron)
+        else:
+            raise ValueError(f'wrong dimension: {self.p["dim"]} is not implemented for problem setup')
+
+    @staticmethod
+    def default_parameters() -> dict[str, pint.Quantity]:
+        """returns a dictionary with required parameters and a set of working values as example"""
+        # this must de defined in each setup class
+
+        setup_parameters = {}
+        setup_parameters['load'] = 10000 * ureg('N/m^2')
+        setup_parameters['length'] = 1 * ureg('m')
+        setup_parameters['height'] = 0.3 * ureg('m')
+        setup_parameters['width'] = 0.3 * ureg('m')  # only relevant for 3D case
+        setup_parameters['dim'] = 3 * ureg('')
```

Comment:
Why do we use units for unitless values?

Reply:
because we force everything in parameter to be a pint object. 
this is the slight drawback, but it think the advantages make up for this.